src/Textures.cpp

```diff
@@ -906,7 +906,7 @@ bool TextureCache::_loadHiresBackground(CachedTexture *_pTexture)
 		params.width = ghqTexInfo.width;
 		params.height = ghqTexInfo.height;
 		params.format = ColorFormatParam(ghqTexInfo.texture_format);
-		params.internalFormat = InternalColorFormatParam(ghqTexInfo.format);
+		params.internalFormat = gfxContext.convertInternalTextureFormat(u32(InternalColorFormatParam(ghqTexInfo.format)));
```

Comment:
In fact it is redundant code. I already did 
ghqTexInfo.format = gfxContext.convertInternalTextureFormat(ghqTexInfo.format);
few lines above.